src/app/overview/overview.component.ts

```diff
@@ -682,6 +713,14 @@ export class OverviewComponent extends SubscriptionManager implements OnInit {
       /[:".,\s()[]{}]/g,
       ''
     );
+
+    if (this.form.value.facetParameter === DimensionName.country) {
+      nvs = nvs.map((nvp: NamesValuePercent) => {
```

Comment:
I've renamed "nvs" to "seriesValues" in this specific case.   Arguably that could be "seriesNameValuePercentValues", but I like succinct names that derive clarity from their context.

I generally find abbreviations to be clear.  The terms "ISO",  "http" and "www" were all invented (as abbreviations) because "extensive" and "clear" are not the same thing, but are often opposing concerns.

The `NameValuePercent` interface describes three values: a name, a value and a percent, and I don't know what could be added to (or subtracted from) that name to make it capture what it does any better.

Reply:
In my opinion, the abbreviations go case by case. The example you gave like 'http', are widely well known, so in those cases it is ok to write the abbreviation, as long as we know what it stands for.
In your code `nvs` it is not clear, at least to me what it stands for, what it represents. Hence why I asked to change the name. But `seriesValues` name seems to be fine.

Regarding `NameValuePercent` name, even though is describes what information it contains, I think it doesn't really say what its encapsulation is supposed to represent, but for now this name is fine.